settings.gradle

```diff
@@ -1,7 +1,7 @@
-rootProject.name = 'candlepin-parent'
-include ':candlepin-common'
-include ':candlepin'
-include ':checks'
-project(':checks').projectDir = "$rootDir/checks" as File
-project(':candlepin-common').projectDir = "$rootDir/common" as File
-project(':candlepin').projectDir = "$rootDir/server" as File
\ No newline at end of file
+rootProject.name = "candlepin-parent"
+include ":candlepin-common"
+include ":candlepin"
+include ":checks"
+project(":checks").projectDir = "$rootDir/checks" as File
+project(":candlepin-common").projectDir = "$rootDir/common" as File
+project(":candlepin").projectDir = "$rootDir/server" as File
```

Comment:
Please add a newline at the end of the file here.

Reply:
Done.